test/ERC20SimpleSwap.should.js

```diff
@@ -176,9 +176,11 @@ function cashChequeInternal(beneficiary, recipient, cumulativePayout, callerPayo
   it('should only emit a chequeBounced event when insufficient funds', function() {
     if(this.totalPayout.lt(cumulativePayout.sub(this.preconditions.paidOut))) {
       expectEvent.inLogs(this.logs, "ChequeBounced", {})
+      expect(this.postconditions.bounced).to.be.true
```

Comment:
You are running this now under the header: `it should emit a chequebounced event`, but it would be more correct to create a separate `it` for this state variable.

Reply:
moved to separate `it`